vochain/scrutinizer/vote.go

```diff
@@ -35,19 +35,19 @@ const kvErrorStringForRetry = "Transaction Conflict"
 
 // GetVoteReference gets the reference for an AddVote transaction.
 // This reference can then be used to fetch the vote transaction directly from the BlockStore.
-func (s *Scrutinizer) GetEnvelopeReference(nullifier []byte) (*VoteReference, error) {
-	txRef := &VoteReference{}
+func (s *Scrutinizer) GetEnvelopeReference(nullifier []byte) (*types.VoteReference, error) {
+	txRef := &types.VoteReference{}
```

Comment:
No need to declare a var here

Reply:
How so? `s.db.FindOne` needs a pointer to a VoteReference to store the query result. 

Are you saying it should use a named return variable?